export_model.py

```diff
@@ -126,7 +126,7 @@ def __call__(self, input_tensor: tf.Tensor) -> MutableMapping[Text, Any]:
       processed_image = tf.concat(
           [processed_image, processed_prev_image], axis=2)
     else:
-      (resized_image, processed_image, _, _, _) = self._preprocess_fn(
+      (resized_image, processed_image, _, _, _, _) = self._preprocess_fn(
```

Comment:
Thanks for the fix, davidblom603!
Could you please also help update Line125?
Thanks,

Reply:
Done